src/mode.js

```diff
@@ -43,59 +40,57 @@ let fullVersion = '';
 
 /**
  * Provides info about the current app.
+ * @param {?Window=} win
```

Comment:
doing that though seems to make it type check more modules which we haven't fixed yet O_O